src/opcodes/tests/test_opcodes.cairo

```diff
@@ -51,3 +51,23 @@ fn test_op_add() {
     let expected_stack = array!["\0\0\0\0\0\0\0\x02"];
     assert_eq!(dstack, expected_stack.span(), "Stack is not equal to expected");
 }
+
+#[test]
+fn test_op_1add() {
+    let program = "OP_1 OP_1ADD";
+    let mut compiler = CompilerTraitImpl::new();
+    let bytecode = compiler.compile(program);
+    let mut engine = EngineTraitImpl::new(bytecode);
+
+    let res = engine.step();
+    assert!(res, "Execution of OP_1 failed");
```

Comment:
Since OP_1 is tested in another function, we can ignore this assertion to keep the tests as minimal as possible

Reply:
@b-j-roberts done.